doc/Motr_Terminology_Guide.rst

```diff
@@ -0,0 +1,172 @@
+| Hierarchical Storage Management (HSM)                   | Hierarchical Storage Management generalizes the concept of data caching. Different storage technologies are organized hierarchically in the tiers from fastest to the slowest.                                                                                                                                                                                                                                                                                                                                                                                                         |
++---------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Key                                                     | An arbitrary sequence of bytes, used to identify a record in a catalogue                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
++---------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
[optional] So many dashes make the diff hard to scroll on mobile phone.  Can we achieve the same rendered document with fewer dashes?

Reply:
This is rst table format. If we remove any dash it will break the table. You can use Preview option to view doc without dashes.